src/styles/blocks/page.scss

```diff
@@ -0,0 +1,17 @@
+.page {
+
+  &__header {
```

Comment:
This element is redundant. You can add this rules in block header, bacause there arn't anyone of external properties.